### PR TITLE
Mobgrade

### PR DESCRIPTION
**Pull request by odra**
## Additional Information

Adds mobile set of products in upgrade playbook.

## Verification Steps

* Install latest 1.4 release tag including backups;
* Switch to this branch and run the upgrade playbook (do not forget to set thos vars to true: `backup_restore_install=True` and `aws_s3_backup_secret_name=s3-credentials`;
* Check if mobile products were installed and mdc login works;
* Check if webapp login works and if mdc is displayed in the left nav;
* Check if manifest file was properly upgraded: `oc get secret/manifest -n  webapp -o jsonpath='{.data.generated_manifest}' | base64 -d` and `oc get secret/manifest -n  mobile-developer-console -o jsonpath='{.data.generated_manifest}' | base64 -d`

## Is an upgrade task required and are there additional steps needed to test this?

- [ ] Tested Installation
- [ ] Tested Uninstallation
- [ ] Tested or Created follow on for Upgrade

### CODE REVIEW

File: playbooks/install_middleware_monitoring_config.yml [hunk 1]
```diff
@@ -24,16 +24,4 @@
...
-      - middleware_monitoring | default(true) | bool
-      - ups | default(true) | bool
-  - include_role:
-      name: mdc
-      tasks_from: monitoring
-    when:
-      - middleware_monitoring | default(true) | bool
-      - mdc | default(true) | bool
\ No newline at end of file
+      - mobile_security_service | default(true) | bool
```
Comment (wei-lee) [inline playbooks/install_middleware_monitoring_config.yml]: should we move the mobile security service moved to the mobile one too?

**Comment by wei-lee** [comment]
@odra I am getting a 1.4.1 cluster now and will try to verify this.

**Comment by wei-lee** [comment]
@odra I have ran the upgrade playbook and it finished successfully. However, I discovered 2 issues:

1. The endpoint monitoring dashboard didn't pick up the new endpoints for all the mobile services. You can see it [here](https://grafana-route-middleware-monitoring.apps.demo-1762.open.redhat.com/d/xtkCtBkiz2/endpoints-detailed?orgId=1&refresh=30s).
2. It looks like backup didn't setup for MSS.

You can use [this cluster](https://master.demo-1762.open.redhat.com) to check. 

Thanks.

**Comment by wei-lee** [comment]
thanks @odra. I have ran it again and I can verify that both issues are now fixed on PDS.

**Comment by psturc** [comment]
Verified the upgrade on OSD and MDC login, but still couldn't log in to solution explorer, since that is fixed in https://github.com/integr8ly/installation/pull/971

**Pull request merged**